csm_web/frontend/static/frontend/css/resource_aggregation.css

```diff
@@ -214,7 +218,9 @@
 	display: flex;
 	flex-direction: row;
 	justify-content: center;
-	margin: 20px auto;
+	margin-top: 20px auto;
+	margin-left: 20px auto;
+	margin-right: 20px auto;
```

Comment:
These can all be combined into one `margin` attribute; the `auto` also doesn't do anything with the CSS if you specify the margin individually.